config/config.go

```diff
@@ -29,10 +30,15 @@ type Config struct {
 	Port uint16 `json:"port"`
 	Debug bool `json:"debug"`
 	Database database `json:"database"`
+	LogFilePath string `json:"logfilepath"`
+	LogFileName string `json:"logfilename"`
 }
 
 var Data Config
 
+func init() {
+	Data.Load(flag.Config)
+}
 func (config *Config) Load(filename string) {
```

Comment:
函数之间要有空行，用 GoLand 执行一下 Code Reformat。
```suggestion
func init() {
	Data.Load(flag.Config)
}

func (config *Config) Load(filename string) {
```